geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -2747,6 +2749,8 @@
   public static final String STATUS_SERVER__DIR = "dir";
   public static final String STATUS_SERVER__DIR__HELP =
       "Working directory in which the Cache Server is running. The default is the current directory.";
+  public static final String STATUS_SERVER__NO_SERVER_SPECIFIED_ERROR_MESSAGE =
```

Comment:
I believe the `running in %2$s` portion of this output is going to be misleading when the status locator command has omitted the `workingDirectory` option.  For example, if I ask for the status of a remote locator by specifying its hostname and port, this message will suggest that we expect the locator to be running in the current directory on the GFSH client machine.  It might also be helpful for this message to explain *why* we are unable to determine the status of this Locator.  Maybe something like  "At least one of the following options must be specified:  `--name`, `--host`, or `--port`.   Use 'help status server' to display detailed usage information." would be more helpful.